### ISSUE TITLE
Need more document on using EffNetV2 without top layer

### ISSUE THREAD

**Issue by riaqn** [google/automl#1055]
Thank you for this (quite complete and polished) project and fixing issues actively.

Apologies if this is a trivial issue. I want to use EffNetV2 for **regression**, so I copied the code from README:
```
model = tf.keras.models.Sequential([
    tf.keras.layers.InputLayer(input_shape=[None, None, 3]),
    effnetv2_model.get_model('efficientnetv2-s', include_top=False, pretrained=True),
    tf.keras.layers.Dropout(rate=0.2),
    tf.keras.layers.Dense(1, activation = None),
])
```
Now do I simply call `model.compile()` and `model.fit()`? There is also `TrainableModel' but it seems quite specific to the dataset format and therefore probably not applicable to me? (In particular, it assumes that `features['image']` is the actual input, as opposed to just `features`) 

In that case, I guess I will just call `model.compile()` and `model.fit()`. I will use similar optimizer/loss/epochs/step_per_epochs because my model is very similar to a classifying EffNetV2. I will use different metrics, of course. The dataset passed to `fit()` also need to conform to the [Keras format ](https://github.com/tensorflow/tensorflow/blob/a4dfb8d1a71385bd6d122e4f27f86dcebb96712d/tensorflow/python/keras/engine/training.py#L765), or I can write my own `train_step` method and such.

Is my understanding correct? Thank you.

**Comment by macsunmood** [comment]
I don't seem to be able to get the model without top (as if `include_top=False` don't work).
Can anyone point out on possible cause/solutions ?
